pytorch_lightning/trainer/distrib_data_parallel.py

```diff
@@ -379,28 +387,30 @@ def spawn_ddp_children(self, model):
         os.environ['NODE_RANK'] = node_rank
         os.environ['LOCAL_RANK'] = '0'
 
+        # when user is using hydra find the absolute path
+        path_lib = abspath if not HYDRA_AVAILABLE else to_absolute_path
+
         # pull out the commands used to run the script and resolve the abs file path
         command = sys.argv
-        full_path = abspath(command[0])
+        try:
```

Comment:
Thanks for adding this.
Can drop a comment here saying the try-catch is a workaround for Hydra 0.11 issue?

Once Hydra 1.0 becomes the minimal supported version this hack can be removed.